agent/discoveryTools.mjs

```diff
@@ -251,13 +264,13 @@ export async function continueSChainDiscoveryInBackgroundIfNeeded( isSilentReDis
                         let strMessage =
                             cc.success( "S-Chain network was re-discovered, " ) +
                             cc.info( cntDiscoveredNew ) +
-                            cc.success( " of " ) + cc.info( cntNodes ) +
+                            cc.success( " of " ) + cc.info( nCountToWait ) +
                             cc.success( " node(s) (" ) + strDiscoveryStatus + cc.success( ")" );
-                        const cntStillUnknown = cntNodes - cntDiscoveredNew;
+                        const cntStillUnknown = nCountToWait - cntDiscoveredNew;
```

Comment:
This could be negative number if the number of already discovered nodes will be more than 2/3